spec/requests/kinesis_spec.rb

```diff
@@ -20,7 +20,7 @@ def http_login(username = Rails.application.secrets.kinesis[:username],
     @env
   end
 
-  it 'processes the stream events' do
+  xit 'processes the stream events' do
```

Comment:
typo

Reply:
I had paused the test because it was annoying me with intermittent failures ( #497 ) and forgot to re-enable it.